src/script/hooks/useRoveFocus.ts

```diff
@@ -24,35 +24,40 @@ import type {KeyboardEvent as ReactKeyboardEvent} from 'react';
 
 import {isKey, isTabKey, KEY} from 'Util/KeyboardUtil';
 
-function useRoveFocus(size: number, defaultFocus = 0, infinite = true) {
```

Comment:
I have simplified the API, since `defaultFocus` and `infinite` were never set to different values